### PR TITLE
[#6667] swiper phases

### PR DESCRIPTION
**Pull request by khamui**

### CODE REVIEW

File: meinberlin/templates/a4modules/includes/module_detail_phase.html [hunk 1]
```diff
@@ -19,25 +19,36 @@
...
-            </div>
-            <div class="phase-info__item__subtitle">
-                {% html_date phase.start_date 'DATETIME_FORMAT' %}
-                &ndash; {% html_date phase.end_date 'DATETIME_FORMAT' %}
-            </div>
-            <div class="phase-info__item__description">
-                {{ phase.description }}
-            </div>
+    <!-- Start Swiper Template -->
+    <div class="swiper-phases swiper">
```
Comment (khamui) [inline meinberlin/templates/a4modules/includes/module_detail_phase.html:23]: only using `swiper-phases` caused problem, so in addition it needs `swiper` to make it work properly.

Comment (phillimorland) [reply meinberlin/templates/a4modules/includes/module_detail_phase.html:6]: @khamui having spoken to @fuzzylogic2000 we do not want to load this when it is a single phase module so we should use a separate include for single phase modules and have this as an include for multi phase modules

Comment (khamui) [reply meinberlin/templates/a4modules/includes/module_detail_phase.html:6]: yes makes sense, will do that

**Review by fuzzylogic2000 — CHANGES_REQUESTED** [review]
When there are multiple modules (in timeline and parallel module view), I get none - none as phase start and end dates. And for the other ones, all phases are still shown on the same swiper tile, but that could be my cache somehow?

**Comment by khamui** [comment]
> When there are multiple modules (in timeline and parallel module view), I get none - none as phase start and end dates. And for the other ones, all phases are still shown on the same swiper tile, but that could be my cache somehow?

@fuzzylogic2000 indeed both bugs were real bugs, so no caching issues. In case of only one phase I am passing `phase` now and to make swiper appear correctly for multiple phases, the styling files have to be loaded before. That should work now.

**Review by fuzzylogic2000 — APPROVED** [review]
Nice! It always works now!

**Pull request merged**